base/levenshtein_dfa.cpp

```diff
@@ -169,10 +180,20 @@ void LevenshteinDFA::State::Normalize()
 
 // LevenshteinDFA ----------------------------------------------------------------------------------
 // static
-LevenshteinDFA::LevenshteinDFA(UniString const & s, size_t prefixCharsToKeep, size_t maxErrors)
+LevenshteinDFA::LevenshteinDFA(UniString const & s, size_t prefixSize,
+                               std::vector<UniString> const & prefixMisprints, size_t maxErrors)
   : m_size(s.size()), m_maxErrors(maxErrors)
 {
   m_alphabet.assign(s.begin(), s.end());
+  CHECK_LESS_OR_EQUAL(prefixSize, s.size(), ());
+  for (auto c = s.begin(); std::distance(c, s.begin()) < prefixSize; ++c)
```

Comment:
Please add a blank line before the `for`

Reply:
done